Dockerfile

```diff
@@ -59,6 +59,9 @@ RUN apt-get update && \
     wget https://www-us.apache.org/dist/maven/maven-3/3.6.1/binaries/apache-maven-3.6.1-bin.tar.gz -P /tmp && \
     tar xf /tmp/apache-maven-*.tar.gz -C /usr/local/opt && \
     ln -s /usr/local/opt/apache-maven-3.6.1 /usr/local/opt/maven
+    mkdir -p ~/.gradle
+    echo "org.gradle.daemon=false" >> ~/.gradle/gradle.properties
```

Comment:
Shouldn't here be single angle bracket for (re)creating file?

Reply:
Actually the file doesn't exists, because of noone has created it at the point.
But if someone do in the future and put some necessary content there - it is not so good idea to remove this content here

Reply:
And if someone adds the same options but with different values?..

Reply:
... So gradle would choose what it prefer on its own :) 
But it is better than remove any other params probably stored in the file.